source/conf.py

```diff
@@ -135,7 +135,7 @@ def build_partials(app, env, docnames):
       partial = open('{}{}{}'.format(source_path, docname, '.rst'), 'r').read()
```

Comment:
@mikeweaver 
I couldn't figure out how to run a command in here like `find . -name "*.tmp" -type f -delete` at the end of the build. (maybe it belongs in the makefile anyhow).
I know they're not part of the repo but it might be nice to clean up the .tmp files when we're done. 
Are they of any use?